app/Http/Controllers/PlanController.php

```diff
@@ -15,11 +16,9 @@ class PlanController extends Controller
      * @param Request $request
      * @return Response
      */
-    public function store(Request $request)
+    public function store(PlanRequest $request)
     {
-        return Plan::create(array_merge($request->all(), [
-            'user_id' => Auth::id(),
-        ]));
+        return Plan::create($request->all());
```

Comment:
Whenever we use a validated request, we need to replace $request->all() with $request->validated()